arches_controlled_lists/migrations/0003_rename_search_only_list_searchable.py

```diff
@@ -0,0 +1,366 @@
+from django.db import migrations
```

Comment:
It may be too late for this PR, but we may be able to avoid diffs like this by using [django-sqlfun](https://pypi.org/project/django-sqlfun/), like we did with pgtrigger in archesproject/arches/pull/12102

Reply:
I mentioned to Johnathan we might want to scrutinize this package a little and not rush it into right now, I'm the only github star and I haven't even tried it ;)